pontoon/sync/formats/l20n.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import absolute_import
+
+import codecs
+import copy
+import logging
+
+from l20n.format.parser import L20nParser
+from l20n.format.serializer import Serializer
+from l20n.format import ast
+
+from pontoon.sync import SyncError
+from pontoon.sync.formats.base import ParsedResource
+from pontoon.sync.vcs.models import VCSTranslation
+
+log = logging.getLogger(__name__)
+
+
+class L20NEntity(VCSTranslation):
+    """
+    Represents entities in l20n (without its attributes).
+    """
+    def __init__(self, type, key, source_string, source_string_plural, strings, comments=None, order=None):
+        self.type = type
```

Comment:
We don't seem to use or need the `type` attribute in L20NEntity, so you can just remove it (and also from the `L20NEntity()` calls of course).